tests/foreman/ui/test_remoteexecution.py

```diff
@@ -61,12 +117,52 @@ def test_positive_create_job_template_input(self):
         8. Populate the template code and reference the newly created input
         9. Click submit
 
-        @Assert: The job template was successfully created
+        @Assert: The job template was successfully saved with new input added
+        """
+        name = gen_string('alpha')
+        var_name = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_job_template(
+                session,
+                name=name,
+                template_type='input',
+                template_content=gen_string('alphanumeric', 20),
+            )
+            self.assertIsNotNone(self.jobtemplate.search(name))
+            self.jobtemplate.add_input(name, var_name)
+            self.jobtemplate.update(
+                name,
+                template_type='input',
+                template_content='<%= input("{0}") %>'.format(var_name)
```

Comment:
this template content should actually look like this ```'<%= input(:{0}) %>'.format(var_name)```
which would result in the template code looking like this
```<%= input(:errata) %>```